tests/config/behat_10.yml

```diff
@@ -11,6 +11,7 @@ default:
               host:                    'onebox'
               vcr_mode:                'none'
               test_site_name:          'behat-tests'
+              test_env_name:           'dev'
```

Comment:
We don't always use dev. This would be best left to the fixtures themselves. Also, a misnomer again. 😛